src/runtime/object.h

```diff
@@ -9,6 +9,8 @@ Author: Leonardo de Moura
 #include <lean/lean.h>
 #include "runtime/mpz.h"
 
+#define EXTERN extern "C" LEAN_EXPORT
```

Comment:
Where are we using `EXTERN`?

Reply:
Ah, I wanted to use this to save some space, but not all runtime cpps include this header, so I didn't use it in the end.